containers/containers.cabal

```diff
@@ -25,15 +25,15 @@ extra-source-files:
     include/containers.h
     changelog.md
 
-tested-with: GHC==9.2.1, GHC==9.0.1, GHC==8.10.7, GHC==8.8.4, GHC==8.6.5, GHC==8.4.4, GHC==8.2.2, GHC==8.0.2, GHC==7.10.3, GHC==7.8.4, GHC==7.6.3
+tested-with: GHC==9.2.1, GHC==9.0.1, GHC==8.10.7, GHC==8.8.4, GHC==8.6.5, GHC==8.4.4, GHC==8.2.2, GHC==8.0.2
 
 source-repository head
     type:     git
     location: http://github.com/haskell/containers.git
 
 Library
     default-language: Haskell2010
-    build-depends: base >= 4.6 && < 5, array >= 0.4.0.0, deepseq >= 1.2 && < 1.5
+    build-depends: base >= 4.9.1 && < 5, array >= 0.4.0.0, deepseq >= 1.2 && < 1.5
```

Comment:
What's the motivation for excluding `base-4.9.0.0` / GHC-8.0.1?

Reply:
I couldn't get the CI set up for it. Dunno what the issue is.

Reply:
Alright. Looks like neither GHCup nor HVR's PPA provide GHC 8.0.1.